api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -181,6 +181,12 @@
      */
     Attribute<Class<?>> TUPLE_CLASS = new Attribute<>(Class2String.getInstance());
 
+    /**
```

Comment:
IMO, it will be better to remove it. There is JIRA, I don't think TODO is necessary.